front-end/src/pages/CreateEvent.jsx

```diff
@@ -1,330 +1,104 @@
-import { useNavigate } from "react-router-dom";
-import { useEffect, useState } from "react";
-import { Container, Row, Col, Button, Form } from "react-bootstrap";
-import { getInterestCategories } from "../utilities/InterestCategoriesUtilities";
-import { postEventDetails } from "../utilities/EventUtilities";
-import LocationSearchMap from "../components/LocationSearchMap";
+import React, { useEffect, useState } from 'react';
```

Comment:
is there a reason why you imported React here?

Reply:
No that's a habit of mine lol ill remove it. if you see it anywhere on the front end it was definitely me.